<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-E36DAG2Q" ignored="0" name="Test1" tests="6" failures="3" timestamp="25 Jun 2019 18:21:28 GMT" time="231.035" errors="0">
  <testcase name="loginTest" time="54.955" classname="com.hybridframework.testcases.TC_LoginTest_001"/>
  <testcase name="loginTest" time="63.377" classname="com.hybridframework.testcases.TC_LoginTest_002"/>
  <testcase name="loginTest" time="4.383" classname="com.hybridframework.testcases.TC_LoginTest_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridframework.testcases.TC_LoginTest_002.loginTest(TC_LoginTest_002.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="4.704" classname="com.hybridframework.testcases.TC_LoginTest_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridframework.testcases.TC_LoginTest_002.loginTest(TC_LoginTest_002.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="loginTest" time="4.743" classname="com.hybridframework.testcases.TC_LoginTest_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.hybridframework.testcases.TC_LoginTest_002.loginTest(TC_LoginTest_002.java:38)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase name="addNewCustomer" time="85.191" classname="com.hybridframework.testcases.TC_AddCustomerTest_003"/>
</testsuite> <!-- Test1 -->
